.buildkite/pipeline.yml

```diff
@@ -0,0 +1,17 @@
+# SPDX-FileCopyrightText: 2021 Arthur Breitman
+# SPDX-License-Identifier: LicenseRef-MIT-Arthur-Breitman
+
+steps:
+  - label: reuse lint
+    commands:
+    - nix run -f ci.nix pkgs.reuse -c
+        reuse lint
+
+  - label: check trailing whitespace
+    commands:
+    - .buildkite/check-trailing-whitespace.sh
+
```

Comment:
Oh, one more thing: it would be nice to have an [xrefcheck](https://github.com/serokell/xrefcheck) verification step too.

Reply:
I've added the xrefcheck step, as well as handled other comment. Let me know if this is good to be merged.